src/core/loader/get-registry.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+import { itemsRegistry, initRegistry } from "../../components/shop/item-registry.js";
+
+export const loadRegistry = (screen, config) => {
+    const registryKeys = getRegistryKeys(config);
+    if (!registryKeys.length) return;
```

Comment:
If there are no items in the config will it return an empty array? Prob a good pattern to do that.

You can delete this line if so.

Reply:
It will, and you're right that the line isn't strictly needed, but early return avoids the need to modify the rest of the loader tests plus I thought it had semantic value. Happy to delete on the 'less is more' principle if you prefer.

Reply:
On reflection, leaving it in is a code smell- it's better to make the needed changes in the loader tests (just adding a mock anyway.)